main/tests/server/src/com/google/refine/tests/grel/GrelTests.java

```diff
@@ -166,6 +166,26 @@ public void testString() throws ParsingException {
             parseEval(bindings, test);
         }
     }
+
+    @Test
+    public void testGetJsonFieldExists() throws ParsingException {
+        String test[] = { "\"[{\\\"one\\\": \\\"1\\\"}]\".parseJson()[0].one", "1" };
+        parseEval(bindings, test);
+    }
+
+    @Test
+    public void testGetJsonFieldAbsent() throws ParsingException {
+        String test =  "\"[{\\\"one\\\": \\\"1\\\"}]\".parseJson()[0].two";
+        Evaluable eval = MetaParser.parse("grel:" + test);
+        Assert.assertNull(eval.evaluate(bindings));
+    }
+
+    @Test
+    public void testGetFieldFromNull() throws ParsingException {
+        String test =  "null.value";
```

Comment:
This should be an error. There is no way to fetch a field from a null.

Reply:
Yes - as noted by @wetneb above. However, this handling for GREL is deliberate following discussion on #820 and #1036